src/Main.ts

```diff
@@ -5,31 +5,31 @@ import * as v8 from "v8";
 import {Client, DIService, SimpleCommandMessage} from "discordx";
 import {Intents, Message} from "discord.js";
 import {moduleRegistrar, registerInstance} from "./DI/moduleRegistrar";
-import {container} from "tsyringe";
+import {container, singleton} from "tsyringe";
 import {GuildManager} from "./model/guild/manager/GuildManager";
 import {SettingsManager} from "./model/settings/SettingsManager";
 import {createConnection, useContainer} from "typeorm";
 import io from "@pm2/io";
 import {importx} from "@discordx/importer";
 import {Settings} from "luxon";
+import {Property} from "./model/decorators/Property";
 // const https = require('http-debug').https;
 // https.debug = 1;
 
-io.init({
-    tracing: true,
-    metrics: {
-        http: true
-    }
-});
-dotenv.config({path: __dirname + '/../.env'});
-
+@singleton()
 export class Main {
     public static testMode = false;
```

Comment:
So if the token_test and token are in env, you might as well put testMode in env too?